### PR TITLE
feat: meshing support

### PR DESCRIPTION
**Pull request by dodamih**

### CODE REVIEW

File: tests/unit/geometry/test_bbox.py [hunk 1]
```diff
@@ -223,6 +225,74 @@ def test_pad_exc(bbox: BBox3D, pad, resolution: Vec3D, expected_exc):
...
+            (1, 3, 5, 3),
+            ValueError,
+        ],
+    ],
+)
+def test_num_splits_exc(bbox: BBox3D, num_splits: Sequence[int], expected_exc):
+    with pytest.raises(expected_exc):
+        bbox.split(num_splits=num_splits)
+
+
```
Comment (supersergiy) [inline tests/unit/geometry/test_bbox.py:295]: Nice tests!

File: zetta_utils/layer/volumetric/constant/backend.py [hunk 2]
```diff
@@ -105,6 +105,9 @@ def get_chunk_size(self, resolution: Vec3D) -> Vec3D[int]:  # pragma: no cover
     def get_dataset_size(self, resolution: Vec3D) -> Vec3D[int]:  # pragma: no cover
         return Vec3D[int](0, 0, 0)
 
+    def get_bounds(self, resolution: Vec3D) -> VolumetricIndex:  # pragma: no cover
+        return VolumetricIndex.from_coords((0, 0, 0), (0, 0, 0), Vec3D[int](1, 1, 1))
```
Comment (supersergiy) [inline zetta_utils/layer/volumetric/constant/backend.py:109]: Not sure if it's important, but maybe it makes sense to make it virtually unbounded -- something life from min to max value of `int64`?

**Review by supersergiy — APPROVED** [review]
Looks good

File: zetta_utils/layer/volumetric/constant/backend.py (hunk 2 above)
Comment (dodamih) [reply zetta_utils/layer/volumetric/constant/backend.py:109]: Good idea - I'll include it in the next PR.

Comment (supersergiy) [reply zetta_utils/layer/volumetric/constant/backend.py:109]: Why not this one?

**Pull request merged**